src/registrar/models/user.py

```diff
@@ -60,6 +61,55 @@ class VerificationTypeChoices(models.TextChoices):
         # after they login.
         FIXTURE_USER = "fixture_user", "Created by fixtures"
 
+    class UserPortfolioRoleChoices(models.TextChoices):
+        """
+        Roles make it easier for admins to look at
+        """
+
+        ORGANIZATION_ADMIN = "organization_admin", "Admin"
+        ORGANIZATION_ADMIN_READ_ONLY = "organization_admin_read_only", "Admin read only"
+        ORGANIZATION_MEMBER = "organization_member", "Member"
+
+    class UserPortfolioPermissionChoices(models.TextChoices):
+        """ """
+
+        VIEW_DOMAINS = "view_domains", "View all domains and domain reports"
+        # EDIT_DOMAINS is really self.domains. We add is hear and leverage it in has_permission
+        # so we have one way to test for portfolio and domain edit permissions
+        # Do we need to check for portfolio domains specifically?
+        # NOTE: A user on an org can currently invite a user outside the org
+        EDIT_DOMAINS = "edit_domains", "User is a manager on a domain"
+
+        VIEW_MEMBER = "view_member", "View members"
+        EDIT_MEMBER = "edit_member", "Create and edit members"
+
+        VIEW_REQUESTS = "view_requests", "View requests"
+        EDIT_REQUESTS = "edit_requests", "Create and edit requests"
```

Comment:
We do have some users  (admin and read only admins) who will be able to view all requests, but not edit the requests they did not make. However, these same people can make and edit requests.  On the flipside, others will follow our normal process where they just see requests they created and can manage those requests. 

With that said, should this be changed to include VIEW_ALL_REQUESTS? Then View_requests could be VIEW_CREATED_REQUESTS. Just to differentiate between the permissions?

Reply:
Same thing on DOMAINS I think?

Reply:
VIEW_MANAGED_DOMAINS vs VIEW_ALL_DOMAINS

Reply:
Yes!